guides/log_collection.md

```diff
@@ -0,0 +1,131 @@
+# Log Collection Guide
+  - [Import Required Modules](#import-required-modules)
+  - [Setup Connection with OpenSearch Cluster](#setup-connection-with-opensearch-cluster)
+  - [Initialize Logger](#initialize-logger)
+  - [Define Custom Handler for OpenSearch](#define-custom-handler-for-opensearch)
+  - [Create OpenSearch Handler and Add to Logger](#create-opensearch-handler-and-add-to-logger)
+  - [Setup Asynchronous Logging Using Queues](#setup-asynchronous-logging-using-queues)
+  - [Clean Up](#clean-up)
+
+# Log Collection Guide
+In this guide, we will look at how to collect logs from your application and send them to OpenSearch.
+
+# Import Required Modules
+Let's import the required modules:
+
+```python
+import urllib3
+urllib3.disable_warnings()
+from datetime import datetime
+import logging
+import queue
+from opensearchpy import OpenSearch
+from logging.handlers import QueueHandler, QueueListener
+```
+
+# Setup Connection with OpenSearch Cluster
```

Comment:
Nit: I would remove the word "cluster", because it's not always a cluster. You could also add a note on how to run something locally with docker, just a 1-liner.